.gitignore

```diff
@@ -4,3 +4,47 @@
 Gemfile.lock
 pkg/*
 
+*.rbc
```

Comment:
Don't just copy-paste a bunch of stuff, only things that are needed. See what is being created during build and exclude those.